Formula/r/recyclarr.rb

```diff
@@ -0,0 +1,33 @@
+class Recyclarr < Formula
+  desc "Automatically sync TRaSH Guides to your Sonarr and Radarr instances"
+  homepage "https://recyclarr.dev"
+  version "7.4.0"
+  url "https://github.com/recyclarr/recyclarr.git", tag: "v#{version}"
```

Comment:
```suggestion
  url "[tarball](https://github.com/recyclarr/recyclarr/archive/refs/tags/v7.4.0.tar.gz)"
  sha256 "xxx"
```

Reply:
I cannot use tarball for this because building my application requires a git repository to properly set up versioning (it uses GitVersion). I do not want to try to work around that if possible. I assume because git repositories are supported by formulas that this is an acceptable solution.

Reply:
I'm not sure what you're asking for. Logs and dump for what?

Reply:
I think tarball will be ok. Then the version parameter is automatically parsed and can be used in this formula. And checksum is required to prevent any unintended action and we wanna check whether it changes by upstream or not.

Reply:
@daeho-ro I appreciate the feedback. I just want to be clear on the requirement. Since my project uses GitVersion, it embeds version information into the application by pulling information from the Git repository it is built within. Below is an example snippet from a generated `GitVersionInformation.g.cs` file. The structure shown contains information pulled from Git. This cannot be generated if I build from a tarball, because there is no git repository available when a build occurs.

Does this make sense?

```cs
static class GitVersionInformation
{
    public const string AssemblySemFileVer = "7.4.1.0";
    public const string AssemblySemVer = "7.4.1.0";
    public const string BranchName = "master";
    public const string BuildMetaData = "";
    public const string CommitDate = "2024-11-16";
    public const string CommitsSinceVersionSource = "4";
    public const string EscapedBranchName = "master";
    public const string FullBuildMetaData = "Branch.master.Sha.40125ee8950bc75b8544fe18298d1dec325f0a70";
    public const string FullSemVer = "7.4.1-dev.4";
    public const string InformationalVersion = "7.4.1-dev.4+Branch.master.Sha.40125ee8950bc75b8544fe18298d1dec325f0a70";
    public const string Major = "7";
    public const string MajorMinorPatch = "7.4.1";
    public const string Minor = "4";
    public const string Patch = "1";
    public const string PreReleaseLabel = "dev";
    public const string PreReleaseLabelWithDash = "-dev";
    public const string PreReleaseNumber = "4";
    public const string PreReleaseTag = "dev.4";
    public const string PreReleaseTagWithDash = "-dev.4";
    public const string SemVer = "7.4.1-dev.4";
    public const string Sha = "40125ee8950bc75b8544fe18298d1dec325f0a70";
    public const string ShortSha = "40125ee";
    public const string UncommittedChanges = "0";
    public const string VersionSourceSha = "4f105f859ad8fd371dbe252062252be07bfd1e17";
    public const string WeightedPreReleaseNumber = "55004";
}
```